src/core/public/core_system.test.ts

```diff
@@ -70,6 +70,11 @@ const defaultCoreSystemParams = {
 beforeEach(() => {
   jest.clearAllMocks();
   MockPluginsService.getOpaqueIds.mockReturnValue(new Map());
+
+  // @ts-expect-error 2339
+  window.performance.mark = jest.fn();
+  // @ts-expect-error 2339
+  window.performance.getEntriesByName = jest.fn().mockReturnValue([]);
```

Comment:
These `@ts-expect-error` directives are not needed

```suggestion
  window.performance.mark = jest.fn();
  window.performance.getEntriesByName = jest.fn().mockReturnValue([]);
```

Reply:
Also, would you mind adding a test where `getEntriesByName` does return something so we can test the `reportEvent` API is called with the right values?

Similarly, would you mind adding these fields to the FTR tests for this event at `test/analytics/tests/instrumented_events/from_the_browser/loaded_kibana.ts`? Essentially, it should be just adding these lines:

```diff
      expect(event.properties.kibana_version).to.be.a('string');
+
+      expect(event.properties.load_start).to.be.a('number');
+      expect(event.properties.bootstrap).to.be.a('number');
+      expect(event.properties.core_create).to.be.a('number');
+      expect(event.properties.setup_done).to.be.a('number');
+      expect(event.properties.start_done).to.be.a('number');

      if (browser.isChromium) {
```

Of, if you want to be super explicit with the tests (about the order in which each mark is called):

```diff
      expect(event.properties.kibana_version).to.be.a('string');
+
+      expect(event.properties.load_start).to.be.a('number');
+      expect(event.properties.bootstrap).to.be.a('number');
+      expect(event.properties.bootstrap).to.be.greaterThan(event.properties.load_start as number);
+      expect(event.properties.core_create).to.be.a('number');
+      expect(event.properties.core_create).to.be.greaterThan(event.properties.bootstrap as number);
+      expect(event.properties.setup_done).to.be.a('number');
+      expect(event.properties.setup_done).to.be.greaterThan(event.properties.core_create as number);
+      expect(event.properties.start_done).to.be.a('number');
+      expect(event.properties.start_done).to.be.greaterThan(event.properties.setup_done as number);

      if (browser.isChromium) {
```